docs/components/encoder/_index.md

```diff
@@ -44,6 +44,262 @@ For configuration information, click on one of the following models:
 | [`incremental`](incremental) | A two phase encoder, which can measure the speed and direction of rotation in relation to a given reference point. |
 | [`single`](single) | A single pin "pulse output" encoder which returns its relative position but no direction. |
 
+## Control your encoder with Viam's client SDK libraries
+
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **code sample** tab, select your preferred programming language, and copy the sample code generated.
+
+When executed, this sample code will create a connection to your robot as a client.
+Then control your robot programmatically by adding API method calls as shown in the following examples.
+
+These examples assume you have an encoder called `"my_encoder"` configured as a component of your robot.
+If your encoder has a different name, change the `name` in the code.
+
+Be sure to import the encoder package for the SDK you are using:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python
+from viam.components.encoder import Encoder
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go
+import (
+  "go.viam.com/rdk/components/encoder"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+The encoder component supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[GetPosition](#getposition) | Get the current position of the encoder.
+[ResetPosition](#resetposition) | Reset the position to zero.
+[GetProperties](#getproperties) | Get the supported properties of this encoder.
+[DoCommand](#docommand) | Send or receive model-specific commands.
+
+### GetPosition
+
+Get the current position of the encoder in ticks or degrees.
+Relative encoders return ticks since last zeroing.
+Absolute encoders return degrees.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `position_type` (PositionType.ValueType): Specify whether to get the current position in ticks (`encoder.PositionType.TICKS`) or in degrees (`encoder.PositionType.DEGREES`).
+  If you are not sure which position type your encoder supports but it is a built-in Viam-supported model, you can leave this parameter unspecified (`encoder.PositionType.UNSPECIFIED`) and it will default to the correct position type.
```

Comment:
same here, `encoder.PositionTypeUnspecified`

Reply:
also, maybe say you can pass in unspecified or pass nothing since the parameter is optional